test/component/ComponentSpec.scala

```diff
@@ -20,13 +20,14 @@ import org.scalatest._
 import org.scalatestplus.play.guice.GuiceOneAppPerTest
 import play.api.Application
 import play.api.inject.guice.GuiceApplicationBuilder
+import util.ExternalServicesConfig.{Host, Port}
 
 abstract class ComponentSpec extends FeatureSpec with GivenWhenThen with Matchers with GuiceOneAppPerTest
   with BeforeAndAfterEach with BeforeAndAfterAll {
 
   override def newAppForTest(testData: TestData): Application = new GuiceApplicationBuilder().configure(Map(
```

Comment:
is this used anywhere?

Reply:
https://github.com/hmrc/api-notification-pull/blob/CDD-1719-code-conventions/test/component/ComponentSpec.scala#L30

https://github.com/hmrc/api-notification-pull/blob/CDD-1719-code-conventions/test/component/ComponentSpec.scala#L31